packages/core/content-manager/server/controllers/relations.js

```diff
@@ -139,7 +139,11 @@ module.exports = {
 
     // searching should be allowed only on mainField for permission reasons
     if (_q) {
-      addFiltersClause(queryParams, { [mainField]: { $containsi: _q } });
+      if(query._m && query._m == 'startsWith') {
+        addFiltersClause(queryParams, { [mainField]: { $startsWithi: _q } });
+      } else {
+        addFiltersClause(queryParams, { [mainField]: { $containsi: _q } });
+      }
```

Comment:
If you think it makes sense to change `_m` then you can reduce this code to just be:

```suggestion
      if(_q) {
        addFiltersClause(queryParams, { [mainField]: { [`$${_m}`]: _q } });
      }
```

This means by default the controller can preserve the default behaviour, but the admin panel requests something different? 🤔